creation/interfaces-and-networks.md

```diff
@@ -692,3 +687,209 @@ documentation](https://github.com/openshift/sriov-network-operator/blob/master/d
+        pciAddress: 0000:65:00.2
+        vendor: 15b3
+        vfID: 0
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.3
+        vendor: 15b3
+        vfID: 1
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.4
+        vendor: 15b3
+        vfID: 2
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.5
+        vendor: 15b3
+        vfID: 3
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.6
+        vendor: 15b3
+        vfID: 4
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.7
+        vendor: 15b3
+        vfID: 5
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:01.0
+        vendor: 15b3
+        vfID: 6
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:01.1
+        vendor: 15b3
+        vfID: 7
+      deviceID: "1015"
+      driver: mlx5_core
+      mtu: 1500
+      name: ens1f0
+      numVfs: 8
+      pciAddress: 0000:65:00.0
+      totalvfs: 8
+      vendor: 15b3
+    - deviceID: "1015"
+      driver: mlx5_core
+      mtu: 1500
+      name: ens1f1
+      pciAddress: 0000:65:00.1
+      totalvfs: 8
+      vendor: 15b3
+    syncStatus: Succeeded
+```
+
+#### Start an SR-IOV VM
+
+Once all the SR-IOV components are deployed, it is needed to indicate how to
+configure the SR-IOV network. Refer to the following
+`NetworkAttachmentDefinition` for an example:
+
+    apiVersion: "k8s.cni.cncf.io/v1"
+    kind: NetworkAttachmentDefinition
+    metadata:
+      name: sriov-net
+      annotations:
+        k8s.v1.cni.cncf.io/resourceName: openshift.io/sriov_net
+    spec:
+      config: '{
+      "type": "sriov",
+      "ipam": {
+        "type": "host-local",
+        "subnet": "10.56.217.0/24",
+        "rangeStart": "10.56.217.171",
+        "rangeEnd": "10.56.217.181",
+        "routes": [{
+          "dst": "0.0.0.0/0"
+        }],
+        "gateway": "10.56.217.1"
+      }
+    }'
+
+Finally, to create a VM that will attach to the aforementioned Network, refer
+to the following VMI spec:
+
+```yaml
+---
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-perf
+  name: vmi-perf
+spec:
+  domain:
+    cpu:
+      sockets: 6
+      cores: 1
+      threads: 1
+      dedicatedCpuPlacement: true
+    memory:
+      hugepages:
+        pageSize: "1Gi"
+    resources:
+      requests:
+        memory: "4Gi"
+      limits:
+        memory: "4Gi"
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      interfaces:
+      - masquerade: {}
+        name: default
+      - name: sriov-net
+        sriov: {}
+      rng: {}
+    machine:
+      type: ""
+  networks:
+  - name: default
+    pod: {}
+  - multus:
+      networkName: default/sriov-mlx  # change me
```

Comment:
No need for change-me if we use name from the SriovNetwork example

Reply:
Done.